qa/SidechainTestFramework/account/smart_contract_resources/package.json

```diff
@@ -22,6 +22,6 @@
     "solhint": "^3.3.7",
     "ts-node": "^10.8.1",
     "typescript": "^4.7.4",
-    "dotenv": "latest"
+    "dotenv": "16.0.3"
```

Comment:
I think this should be `^10.0.3`, aka. "compatible with 10.0.3" not "exactly 10.0.3".

Reply:
@akrckn let's do this change and then I will merge it